### ISSUE TITLE
sink kafka support Kerberos

### ISSUE THREAD

**Issue by michael-oou** [loggie-io/loggie#127]
releases v1.0.0 can not connect to kafka with Kerberos authentication

**Comment by ethfoo** [comment]
yep, We use the [kafka-go](github.com/segmentio/kafka-go) as Kafka client in Loggie, but unfortunately it does not support kerberos for now. So we are considering adding another kafka client, maybe [sarama](https://github.com/Shopify/sarama)? I'm just working on it.

**Comment by yuzhichang** [comment]
@ethfoo I think franz-go is the best one currently.
